global.json

```diff
@@ -1,16 +1,16 @@
 {
   "sdk": {
-    "version": "9.0.100-alpha.1.23568.4"
+    "version": "9.0.100-alpha.1.23577.5"
   },
   "tools": {
-    "dotnet": "9.0.100-alpha.1.23568.4",
+    "dotnet": "9.0.100-alpha.1.23577.5",
     "runtimes": {
       "dotnet/x64": [
-        "8.0.0-rtm.23516.15",
+        "8.0.0",
```

Comment:
It doesn't look correct as the tests are failing:
```
Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
Blame: Attaching crash dump utility to process testhost (2612).
Testhost process for source(s) 'D:\a\_work\1\s\artifacts\bin\Microsoft.AspNetCore.AsyncState.Tests\Release\net8.0\Microsoft.AspNetCore.AsyncState.Tests.dll' exited with error: You must install or update .NET to run this application.
App: D:\a\_work\1\s\artifacts\bin\Microsoft.AspNetCore.AsyncState.Tests\Release\net8.0\testhost.exe
Architecture: x64
Framework: 'Microsoft.NETCore.App', version '8.0.0' (x64)
.NET location: D:\a\_work\1\s\.dotnet
The following frameworks were found:
  9.0.0-alpha.1.23576.1 at [D:\a\_work\1\s\.dotnet\shared\Microsoft.NETCore.App]
  9.0.0-alpha.1.23576.3 at [D:\a\_work\1\s\.dotnet\shared\Microsoft.NETCore.App]
Learn more:
https://aka.ms/dotnet/app-launch-failed
To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=8.0.0&arch=x64&rid=win-x64&os=win10
. Please check the diagnostic logs for more information.

Test Run Aborted.
```

Reply:
It wasn't correct before either. I don' know what kind of version this is requiring, do you?

Reply:
Restarting the build as your comment was done while GH had issue and this version change didn't trigger the builds so I actually don't know if that works.

Reply:
It worked.

Reply:
Sweet!